mnist/mlcube.yaml

```diff
@@ -1,11 +1,24 @@
-schema_version: 1.0.0
-schema_type: mlcube_root
-
 name: mnist
-author: MLPerf Best Practices Working Group
-version: 0.1.0
-mlcube_spec_version: 0.1.0
+description: MLCommons MNIST MLCube example
+authors:
+  - {name: "Sergey Serebryakov", email: "sergey.serebryakov@hpe.com", org: "Hewlett-Packard Labs"}
+
+platform:
```

Comment:
Can we will out all fields here?

e.g. 
   accelerator_maker: NVIDIA
   accelerator_model: A100-80GB
   accelerator_count: 8
   host_memory_gb: 40
   need_internet_access: True
   host_disk_space_gb: 100